src/views/LoggingRoom/js/logging-room-create.js

```diff
@@ -64,7 +59,12 @@ export default {
       if (this.titleTemp && this.descriptionTemp && this.members.length) {
         if (!this.isEdit) {
           loggingRoomApi.createLoggingRoom(response => {
-            this.$toasted.success('created')
+            this.toast({
+              data: {
+                message: 'success created logging room',
```

Comment:
`success -> Success`, check again on the rest of this file

Reply:
Resolved, please check again